src/app/_services/api/admin/question-count.service.ts

```diff
@@ -0,0 +1,31 @@
+import {Injectable} from '@angular/core';
```

Comment:
There is no need to have 2 separate files for one api endpoint. You should have 1 file (e.g admin.service.ts) and have both functions in that file.

Reply:
I've combined both services into admin.service.ts